@navikt/core/react/src/form/combobox/combobox.stories.tsx

```diff
@@ -461,12 +461,7 @@ export const InModal: StoryFn = () => {
   return (
     <>
       <Button onClick={() => modalRef.current?.showModal()}>Åpne modal</Button>
-      <Modal
-        ref={modalRef}
-        header={{ heading: "Overskrift" }}
-        width="medium"
-        style={{ height: "auto" }}
-      >
+      <Modal ref={modalRef} header={{ heading: "Overskrift" }} width="medium">
         <Modal.Body style={{ height: "100% " }}>
```

Comment:
```suggestion
        <Modal.Body>
```
Causes bug in Safari (and isn't needed anyways)

Reply:
Took the liberty to fix this. Also added a new story for testing.